### PR TITLE
Update CHANGELOG and VERSION for v0.8 release

### PR DESCRIPTION
**Pull request by elton-furtado**

### CODE REVIEW

File: CHANGELOG/CHANGELOG-0.8.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Changelog 0.8
+
+## 0.8.0 - 2023-10-26
+
+### Changed
+
+- Updated the Grafana docker image for future plugins. ([#524](https://github.com/antrea-io/theia/pull/524), [@Dhruv-J])
```
Comment (antoninbas) [inline CHANGELOG/CHANGELOG-0.8.md:7]: Remove "for future plugins". As far as users are concerned, the version of Grafana we install as part of Theia has simply been updated to a more recent version.

File: CHANGELOG/CHANGELOG-0.8.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+- Updated the node version and toolkit to create plugin ([#536](https://github.com/antrea-io/theia/pull/536), [@yuntanghsu])
```
Comment (antoninbas) [inline CHANGELOG/CHANGELOG-0.8.md:8]: s/and toolkit to create plugin/and replaced toolkit with create-plugin for plugin development

File: CHANGELOG/CHANGELOG-0.8.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+- Changed ClickHouse volume ownership. ([#557](https://github.com/antrea-io/theia/pull/557), [@yanjunz97])
+
+
+### Fixed
+
+- Fixed the allow namepsace list in policy recommendation job. ([#418](https://github.com/antrea-io/theia/pull/418), [@dreamtalen])
```
Comment (antoninbas) [inline CHANGELOG/CHANGELOG-0.8.md:14]: s/namepsace/namespace

**Review by tnqn — APPROVED** [review]
LGTM

**Pull request merged**